apps/ledger-live-desktop/tests/specs/speculos/subAccount.spec.ts

```diff
@@ -0,0 +1,113 @@
+import test from "../../fixtures/common";
+import { Token } from "../../enum/Tokens";
+import { specs } from "../../utils/speculos";
+import { Application } from "tests/page";
+import { addTmsLink } from "tests/fixtures/common";
+
+const tokens: Token[] = [
+  Token.ETH_USDT,
+  //Token.XLM_USCD, //TODO: Reactivate when Date.Parse issue is fixed - desactivate time machine for Speculos tests
+  Token.ALGO_USDT,
+  Token.TRON_USDT,
+  Token.BSC_BUSD,
+  Token.MATIC_DAI,
+];
+
+const tokensReceive: Token[] = [
+  Token.ETH_USDT,
+  Token.ETH_LIDO,
+  Token.TRON_USDT,
+  Token.TRON_BTT,
+  Token.BSC_BUSD,
+  Token.BSC_SHIBA,
+  Token.MATIC_DAI,
+  Token.MATIC_UNI,
+];
+
+for (const [i, token] of tokens.entries()) {
+  test.describe.parallel("Add subAccount without parent @smoke", () => {
+    test.use({
+      userdata: "skip-onboarding",
+      testName: `add subAccount without parent (${token.tokenName})`,
+      speculosCurrency: specs[token.parentAccount.currency.deviceLabel.replace(/ /g, "_")],
+      speculosOffset: i,
+    });
+
+    test(`Add Sub Account with parent (${token.parentAccount.currency.uiName})`, async ({
+      page,
+    }) => {
+      addTmsLink(["B2CQA-2448"]);
+
+      const app = new Application(page);
+
+      await app.portfolio.openAddAccountModal();
+      await app.addAccount.expectModalVisiblity();
+
+      await app.addAccount.selectToken(token);
+      await app.addAccount.addAccounts();
```

Comment:
In the ticket it says`Info box should be displayed`. I think we are missing this check here

Reply:
done